pyt/stmt_visitor_helper.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-CALL_IDENTIFIER = '¤'
+CALL_IDENTIFIER = '~'
```

Comment:
So I had to change this to `~` because the other symbol was unicode and showed up as like `\u0358` in the json output, so you can ignore the changes to the tests because they more or less just have to do with me changing this across all files.